### PR TITLE
Omni wheel and yaml adaptations

### PR DESCRIPTION
**Pull request by olgen2013**
For the adaptations in cob_drive on branch "Omni wheel and yaml adaptations" is a new yaml-config requiered. This yaml-config-file was added and a new base_controller.launch which starts the adaptet undercarriage_control ros-node.

These adapatations were arranged with @ipa-mig and @ipa-mdl

### CODE REVIEW

**Comment by mathias-luedtke** [comment]
- needs https://github.com/ipa320/cob_control/pull/23 and https://github.com/ipa320/cob_driver/pull/206
- yaml format is sufficient for now, but will need some tweaks later on to be useable with the plugin controller
- commit history should be squashed or merged manually since it includes a lot of testing commits

**Comment by floweisshardt** [comment]
visual review: OK

File: cob_bringup/robots/raw3-1.xml [hunk 1]
```diff
@@ -80,6 +80,10 @@
 		<include file="$(find cob_bringup)/drivers/openni.launch" >
 			<arg name="robot" value="$(arg robot)" />
 		</include>
+                <include file="$(find cob_bringup)/tools/teleop.launch" >
+                        <arg name="robot" value="$(arg robot)" />
+                        <arg name="version" value="v1" />
+                </include>
```
Comment (mgruhler) [inline cob_bringup/robots/raw3-1.xml:86]: why has teleop been moved to PC1 instead of leaving it at PC2?

**Comment by mgruhler** [comment]
as cob_control#23 is superseded by ipa320/cob_control#24, this should not be merged until cob_control#24 and cob_driver#206 have been tested and merged, respectively.
Also, due to new YAML format in cob_control#24, this PR would (again) need to be adapted accordingly.

**Comment by mgruhler** [comment]
@ipa-mdl @ipa-fmw @olgen2013 
due to adaptations in ipa320/cob_control#24 and the decision to not merge ipa320/cob_driver#206, this PR is outdated and can be closed!

**Pull request closed without merging**